app/models/movie.rb

```diff
@@ -44,6 +44,9 @@ def self.by_tag_and_user(tag, user)
   scope :by_longest_runtime, -> { order(:runtime).reverse_order }
   scope :by_recent_release_date, -> { order(:release_date).reverse_order }
   scope :by_highest_vote_average, -> { order(:vote_average).reverse_order }
+  scope :default_list_order, -> (list) do
+    list.movies.order('listings.created_at desc, listings.priority desc')
+  end
```

Comment:
Just writing this down since we just discussed it...
This secondary sort won't be effective against a timestamp. If we'd like to have this feature work, we'd have to process the `created_at` a little bit, like `listings.created_at.to_date` to make it possible for an entry to have the same date as another, therefore allowing space for the priority ranking.